configuration/mobile/overview.mdx

```diff
@@ -24,18 +24,18 @@ When a user clicks on a universal link, the operating system checks whether the
 Universal links offer several benefits for both users and developers:
 
 - **Simplified user experience** - By directing users to the relevant content or app store, universal links create a more streamlined and intuitive user journey, which can lead to increased user engagement and retention.
-- Improved conversion rates** - By reducing friction in the user experience and lowering the barriers to entry, universal links can help boost conversion rates for app installations that lead to user sign-ups.
+- **Improved conversion rates** - By reducing friction in the user experience and lowering the barriers to entry, universal links can help boost conversion rates for app installations that lead to user sign-ups.
 - **Enhanced security** - Universal links are more secure than legacy custom URL schemes, as they rely on the app's domain and can only be used by the associated app, thus reducing the risk of phishing attacks or unauthorized access.
 
 ### Configuration
 
 To enable mobile app authentication, you must first configure your mobile app details in the Rownd dashboard.
 
 1. Select **Sign-in methods** from the left navigation menu.
-2. In the Sign-in preferences section, locate **Mobile app sign-in**
+2. In the Sign-in preferences section, locate **Mobile app settings**
 3. Press **Enable**.
 
-![Configuring mobile apps only takes a few minutes.](/images/mobile-app-settings.png)
+![Configuring mobile apps only takes a few minutes.](/images/mobile-app-settings2.png)
```

Comment:
It might be worth linking to the Android and iOS pages here. Like, "See our step-by-step configuration guides for [Android] and [iOS] for assistance in filling out this section.

Reply:
Also, is the information lower down in this file redundant now that we have pages specific to Android and iOS?

Reply:
Yup I agree with that. I removed the duplicated content and put in a Card group for links to Android and iOS configuration.